import-export-cli/utils/constants.go

```diff
@@ -168,3 +168,9 @@ const DynamicEndpointConfig = `{"endpoint_type":"default","sandbox_endpoints":{"
 const AwsLambdaRoleSuppliedAccessMethod = "role_supplied"        // To denote "accessMethod: role_supplied" in api_params.yaml
 const AwsLambdaRoleSuppliedAccessMethodForJSON = "role-supplied" // To denote the "accessMethod : role-supplied" in api.json
 const AwsLambdaStoredAccessMethod = "stored"                     // To denote "accessMethod: stored" in api_params.yaml and to denote the "accessMethod : stored" in api.json
+
+const MockTestExportDirectory ="testdata/MockExportDirectory"
+const TestSwagger2DefinitionPath = "testdata/swagger2Definition.yaml"
+const TestOpenAPI3DefinitionPath = "testdata/openAPI3Definition.yaml"
```

Comment:
Is it possible to use a different constant file to define test-related variables?